.github/workflows/pd-tests.yaml

```diff
@@ -0,0 +1,92 @@
+on:
+  push:
+      branches:
+        - master
+        - release-4.0
+  pull_request:
+    branches:
+      - master
+      - release-4.0
+name: PD Tests
+jobs:
+  chunks:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        worker_id: [1, 2, 3, 4]
+    outputs:
+        job-total: ${{ strategy.job-total }}
+    steps:
+    - uses: actions/setup-go@v2
+      with:
+        go-version: 1.14
+    - name: Checkout code
+      uses: actions/checkout@v2
+    - name: Restore cache
+      uses: actions/cache@v2
+      with:
+        path: |
+          ~/go/pkg/mod
+          ~/.cache/go-build
+          ~/work/pd/pd/.tools
+          ~/work/pd/pd/.dashboard_asset_cache
+        key: ${{ runner.os }}-go-${{ matrix.worker_id }}-${{ hashFiles('**/go.sum') }}
+        restore-keys: |
+          ${{ runner.os }}-go-${{ matrix.worker_id }}
+          ${{ runner.os }}-go-
+    - name: Dispatch Packages
+      id: packages-units
+      env:
+        WORKER_ID: ${{ matrix.worker_id }}
+        TYPE: ${{ matrix.type }}
+      run: |
+        go list ./... |grep -v -E  "github.com/tikv/pd/tests/client|github.com/tikv/pd/server/api" > packages.list
+        total=$(expr ${{ strategy.job-total }} - 1)
+        echo  "Dispatched ${total} normal chunks"
+        split packages.list -n r/${total} packages_unit_ -a 1 --numeric-suffixes=1
+        echo  "Dispatched 2 special task to the last chunk(the special test takes a long time)"
+        echo github.com/tikv/pd/tests/client > packages_unit_${{ strategy.job-total }}
+        echo github.com/tikv/pd/server/api >> packages_unit_${{ strategy.job-total }}
+        packages="{$(cat packages_unit_${WORKER_ID} |tr "\n" ",")}"
+        echo "This worker will test the chunk - ${packages}"
+        echo "::set-output name=list::${packages}"
+    - name: Make Test
+      env:
+        WORKER_ID: ${{ matrix.worker_id }}
+      run: |
+        make test-with-cover TEST_PKGS="${{ steps.packages-units.outputs.list }}"
```

Comment:
Compared with the previous `make ci`, is it missing some targets?

Reply:
The targets include `make ci`, which is listed by `go list ./...`.  Here it is equivalent to `make test`, we need it to calculate the code coverage.